pkg/scheduler/testdata/frontend_to_scheduler_k173.bin

```diff
@@ -0,0 +1,3 @@
+*"100*
+
+foobar"Hello echo!
```

Comment:
I don't see this or the other `querier_k173.bin` being used? if we keep them we should maybe also name them to something more generic instead of keeping the `k173` designation?